lib/elm-test.js

```diff
@@ -302,15 +302,15 @@ if (args._[0] === "make") {
         ? 'No tests found for the file pattern "' +
           fileGlobs.toString() +
           '"\n\nMaybe try running elm-test with no arguments?'
-        : "No tests found in the tests/ directory.\n\nNOTE: Make sure you're running elm-test from your project's root directory, where its elm.json lives.\n\nTo generate some initial tests to get things going, run elm-test init";
+        : "No tests found in the tests/ directory.\n\nNOTE: Make sure you're running elm-test from your project's root directory, where its elm.json lives.\n\nTo generate some initial tests to get things going, run elm-test init.\n\nAlternatively, if you have tests in a different directory, try calling elm-test with a glob: elm-test 'frontend-app/**/*Tests.elm'.";
```

Comment:
Maybe say `"Alternatively, if your application has tests"` instead? This advice won't work if they are working on a package!

Reply:
Nice idea - updated 👍